cl/opinion_page/templates/publish.html

```diff
@@ -0,0 +1,121 @@
+{% endblock %}
+{% block og_title %}
+  Decisions for Tennessee Workers' Compensation Courts – CourtListener.com
+{% endblock %}
+{% block description %}
+  Search and read decisions from the Tennessee Court of Workers' Compensation Claims and Tennessee Workers' Compensation Appeals Board
+{% endblock %}
+{% block og_description %}
+  Search and read decisions from the Tennessee Court of Workers' Compensation Claims and Tennessee Workers' Compensation Appeals Board
+{% endblock %}
+
+{% block navbar-o %}active{% endblock %}
+{% block sidebar %}{% endblock %}
+
+{% block head %}
+  {% if DEBUG %}
+    <link rel="stylesheet"
+          href="{{ STATIC_URL }}css/bootstrap-datepicker3.css"/>
+  {% else %}
+    <link rel="stylesheet"
+          href="{{ STATIC_URL }}css/bootstrap-datepicker3.min.css"/>
+  {% endif %}
+{% endblock %}
+
+{% block footer-scripts %}
+  {% include "includes/date_picker.html" %}
+  <script src="{{ STATIC_URL }}js/jquery.bootstrap-growl.js"></script>
+{% endblock %}
+
+{% block content %}
+  <div id="court_publish" class="col-xs-12">
+    <div class="row">
+      <div class="col-xs-1"></div>
+      <div class="col-xs-2">
+        <img src="{{ STATIC_URL }}img/tn-court.png">
+      </div>
+      <div class="col-xs-8">
+        <h1>
+          {{ form.initial.court }} Upload Portal
+        </h1>
+        <div class="lead gray">
+          <p>Upload court opinions using the form below.</p>
+        </div>
+      </div>
+      <div class="col-xs-1"></div>
+    </div>
+
+  <div class="row">
+    <div class="col-xs-12">
+
+      {% if form.errors %}
+        {% for field in form %}
+          {% for error in field.errors %}
+            <div class="alert alert-danger">
+              <strong>{{ error|escape }}</strong>
+            </div>
+          {% endfor %}
+        {% endfor %}
+        {% for error in form.non_field_errors %}
+            <div class="alert alert-danger">
+                <strong>{{ error|escape }}</strong>
+            </div>
+        {% endfor %}
+      {% endif %}
+
+      <hr><br>
+
+      <form method="post" action=""
+              enctype="multipart/form-data"
+              class="form-horizontal" style="">
+        {% csrf_token %}
+
+        {% for field in form.hidden_fields %}
+          {{ field }}
+        {% endfor %}
+
+        {% for field in form.visible_fields %}
+
+          {% if "Cite" not in field.label %}
+
+            <div class="form-group">
+              <label for="{{ field.name }}" class="col-sm-2 control-label">{{ field.label }}</label>
+              <div class="col-sm-9">
+                {{ field }}
+              </div>
+            </div>
+
+          {% else %}
+
+            {% if "Cite Year" in field.label %}
+              <div class="form-group">
+                <label for="{{ field.name }}" class="col-sm-2 control-label">Neutral Citation</label>
+            {% endif %}
+
+              <div class="col-sm-3">
+                {{ field }}
+              </div>
```

Comment:
Can we make the reporter field a hidden field, and then just put the value of the field here instead so it's not a drop down? It'd look like:

```
Neutral citation:     [Year     ]  TN CT Ap.  [Number    ]
```

Reply:
yes

Reply:
instead of messing with a hidden field why dont we just disable the dropdown.